cmd/apiserver/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/BurntSushi/toml"
+	"github.com/go-park-mail-ru/2023_1_BKS/internal/app/apiserver"
+)
+
+var (
+	configPath string
+)
+
+func init() {
+	flag.StringVar(&configPath, "config-path", "configs/apiserver.toml", "path to config file")
+}
+
+func main() {
+	flag.Parse()
+
+	config := apiserver.NewConfig()
+	_, err := toml.DecodeFile(configPath, config)
+	if err != nil {
+		log.Fatal("Error to decode file: ", err)
+	}
+	// Запускаем сервер
+	if err := apiserver.Start(config); err != nil {
+		log.Fatal(err)
+	}
+
```

Comment:
Пресловутые комменты ради комментов)

```suggestion
	
	if err := apiserver.Start(config); err != nil {
		log.Fatal(err)
	}
```

Комменты в коде нужны либо ради документации пакета, либо в нетривиальных кусках кода, где они помогают разобраться. А тут и так написано, что сервер стартует